app/store/keys/keys.reducer.ts

```diff
@@ -56,3 +56,7 @@ export const createKeysReducer = (keys: Partial<KeysState> = {}) =>
 export const selectKeysSlice = (state: RootState) => state.keys;
 
 export const selectMnemonic = createSelector(selectKeysSlice, state => state.mnemonic);
+export const selectAddress = createSelector(selectKeysSlice, state => {
```

Comment:
Interesting approach - I could see this becoming problematic, but for the screens I can think of for this project, this is probably a good solution.

Reply:
Are you referring to the throw?

Reply:
Yes - should have used better line numbers in my comment